listenbrainz/webserver/static/css/rc-slider.less

```
@@ -0,0 +1,298 @@
+// Converted rc-slider/assets/index.css to .less
+
+// TODO name colors properly
+@color_1: #999;
+@color_2: #666;
+@slider_base_color: #8d8d8d;
+@background_color_2: #abe2fb;
+@background_color_4: #ccc;
+@background_color_5: #6c6c6c;
+@border_color_1: @orange;
+@border_color_2: @orange;
+@border_color_3: #96dbfa;
+@border_color_4: #ccc;
+@border_color_5: transparent;
+@border_top_color_1: #6c6c6c;
+
+@keyframes rcSliderTooltipZoomDownIn {
+  0% {
+    transform: scale(0, 0);
+    transform-origin: 50% 100%;
+    opacity: 0;
+  }
+  100% {
+    transform: scale(1, 1);
+    transform-origin: 50% 100%;
+  }
+}
+@keyframes rcSliderTooltipZoomDownOut {
+  0% {
+    transform: scale(1, 1);
+    transform-origin: 50% 100%;
+  }
+  100% {
+    transform: scale(0, 0);
+    transform-origin: 50% 100%;
+    opacity: 0;
+  }
+}
+.rc-slider {
+  position: relative;
+  width: 100%;
+  height: 14px;
+  padding: 5px 0;
+  border-radius: 6px;
+  touch-action: none;
+  box-sizing: border-box;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  * {
+    box-sizing: border-box;
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  }
+}
+.rc-slider-rail {
+  position: absolute;
+  width: 100%;
+  height: 6px;
+  background-color: @slider_base_color;
+  border-radius: 6px;
+
+  &:hover ~ .rc-slider-handle {
+    border: solid 5px lighten(@orange, 10%);
+  }
+}
+.rc-slider-track {
+  position: absolute;
+  height: 6px;
+  background-color: @background_color_2;
+  border-radius: 6px;
+}
+.rc-slider-handle {
+  position: absolute;
+  width: 3rem;
+  height: 3rem;
+  margin-top: -12px;
```

I would make this quite a bit smaller (at least on desktop sizes):
```suggestion
  width: 2rem;
  height: 2rem;
  margin-top: -7px;
```

---

Was wondering the same few days ago. Will be done :)